src/object/obj_internal.h

```diff
@@ -202,4 +202,9 @@ obj_dkey2hash(daos_key_t *dkey)
 			       dkey->iov_len, 5731);
 }
 
+/* obj_class.c */
+int daos_oc_gftbls_map_init();
```

Comment:
(style)  Bad function definition - int daos_oc_gftbls_map_init() should probably be int daos_oc_gftbls_map_init(void)